src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -92,6 +92,9 @@
 
          Available options:
          "reboot" (type 'b')
+         "install-packages" (type 'as')
+         "uninstall-packages" (type 'as')
+         "install-local-packages" (type 'ah')
```

Comment:
Interesting.  The downside of this is that it would require pushing all of the logic for computing the delta to the client.   If I just want to add a package, I need to find out what the current value of `regenerate-initramfs` is and pass that in.

Now, perhaps we could prefix all of the values with `m`.  But even then, for e.g. the case of installing a package the client would need to the the *current* array and extend it.